src/renderer/lib/playlist.js

```diff
@@ -0,0 +1,161 @@
+var TorrentPlayer = require('./torrent-player')
+
+module.exports = Playlist
+
+function Playlist (torrentSummary) {
+  this._infoHash = torrentSummary.infoHash
+  this._position = 0
+  this._tracks = extractTracks(torrentSummary)
+  this._order = range(0, this._tracks.length)
+
+  this._repeat = false
+  this._shuffled = false
+}
+
+// =============================================================================
```

Comment:
I wouldn't add ascii banners like this. We don't have them anywhere else in the code